src/app/public/modules/date-range-picker/fixtures/date-range-picker.fixture.html

```diff
@@ -0,0 +1,14 @@
+<div>
+  <form
+    novalidate
+  >
+    <sky-date-range-picker
+      name="donation"
+      [(ngModel)]="value"
```

Comment:
style: 2 way binding should come after inputs.

Reply:
ah yeah, nice